### PR TITLE
Update scrollToRow value, when user scrolled list

### PR DESCRIPTION
**Pull request by AleshaOleg**
I tested with passing `isKeyboardNavigationEnabled` prop for list in "People" tab.
What I did, is updating the selected item (which is supposed to be selected with keyboard navigation) when the user scrolling.

Behavior not perfect, but still much much better than it was.
As the user most likely stopped scrolling somewhere in the middle of the item, so the first item from visible items not fully visible. That's why a small jump occurs when the user stops scrolling.

### CODE REVIEW

File: src/components/list/InfiniteAutoRowHeightList.js [hunk 1]
```diff
@@ -21,50 +21,70 @@ const InfiniteAutoRowHeightList = memo(
...
+                    overscanRowCount={overscanRowCount}
+                    onRowsRendered={onRowsRendered}
+                    rowRenderer={({
+                      index,
+                      key,
+                      isVisible,
+                      isScrolling,
+                      style,
+                    }) => {
+                      if (
```
Comment (AleshaOleg) [inline src/components/list/InfiniteAutoRowHeightList.js:67]: This `if` condition basically the only change I did in the list logic

File: src/components/list/InfiniteAutoRowHeightList.js [hunk 2]
```diff
@@ -21,50 +21,70 @@ const InfiniteAutoRowHeightList = memo(
...
+                        isKeyboardNavigationEnabled &&
+                        isVisible &&
+                        !isScrolling
+                      ) {
+                        keyStepperRef.current.setScrollIndexes({
+                          scrollToRow: index,
```
Comment (sk7) [inline src/components/list/InfiniteAutoRowHeightList.js:73]: now it doesn't jump to the first user but still jumps.

as soon as is top scrolling, I think it scrolls the the last visible user currently, which is almost never exactly at the scroll position where the code wants it to be, so this looks like a confusing jump.

**Review by sk7 — CHANGES_REQUESTED** [review]
I'm not sure how to fix it but it still jumps, see other comment

**Comment by AleshaOleg** [comment]
@sk7 updated PR, now updating `scrollToIndex` property for List manually and only when a user using keyboard navigation. Works correctly now.

**Comment by sk7** [comment]
Note: keyboard navigation is quite slow, but still merging this

**Pull request merged**